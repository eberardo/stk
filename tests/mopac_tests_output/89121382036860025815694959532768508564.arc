

                     SUMMARY OF PM7 CALCULATION, Site No: 23169

                                                       MOPAC2016 (Version: 16.175M)
                                                       Wed May 10 12:35:59 2017
                                                       No. of days remaining =  45

           Empirical Formula: C12 H12 N2  =    26 atoms

PM7 NOOPT  EPS=80.1  CHARGE=1
 89121382036860025815694959532768508564



     NO PARAMETERS MARKED FOR OPTIMIZATION, SO 1SCF WAS USED  
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        156.30008 KCAL/MOL =     653.95955 KJ/MOL
          TOTAL ENERGY            =      -2003.35696 EV
          ELECTRONIC ENERGY       =      -2922.56256 EV
          CORE-CORE REPULSION     =        919.20560 EV
          DIELECTRIC ENERGY       =         -2.69325 EV
          DIPOLE                  =         15.33153 DEBYE    POINT GROUP:       C2  
          (SZ)                    =          0.500000
          (S**2)                  =          0.963000
          NO. OF ALPHA ELECTRONS  =         35
          NO. OF BETA  ELECTRONS  =         34
          CHARGE ON SYSTEM        =          1
          IONIZATION POTENTIAL    =          8.726065 EV
          ALPHA SOMO LUMO (EV)    =         -8.726 -0.971
          BETA  SOMO LUMO (EV)    =         -8.919 -2.595
          MOLECULAR WEIGHT        =        184.2402
          COSMO AREA              =        225.62 SQUARE ANGSTROMS
          COSMO VOLUME            =        232.49 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    16    H    21    10.92219
            H    17    H    23     4.77163
            H    26    H    18     2.20942
          SCF CALCULATIONS        =          2
          WALL-CLOCK TIME         =          0.043 SECONDS
          COMPUTATION TIME        =          0.171 SECONDS


          FINAL GEOMETRY OBTAINED
PM7 NOOPT  EPS=80.1  CHARGE=1
 89121382036860025815694959532768508564

  N     1.36200000 +0   1.52200000 +0   4.79600000 +0
  C     0.88600000 +0   1.39700000 +0   3.55000000 +0
  C     0.06200000 +0   0.27200000 +0   3.19600000 +0
  C    -0.36000000 +0   0.11900000 +0   1.89700000 +0
  C     0.00400000 +0   1.05500000 +0   0.90100000 +0
  C    -0.42200000 +0   0.85500000 +0  -0.47600000 +0
  C    -1.77300000 +0   0.62900000 +0  -0.77100000 +0
  C    -2.19300000 +0   0.47200000 +0  -2.08000000 +0
  C    -1.25100000 +0   0.51700000 +0  -3.12800000 +0
  N    -1.65500000 +0   0.28200000 +0  -4.43800000 +0
  C     0.11500000 +0   0.71200000 +0  -2.83300000 +0
  C     0.51500000 +0   0.88800000 +0  -1.51900000 +0
  C     0.77700000 +0   2.18400000 +0   1.25100000 +0
  C     1.20000000 +0   2.37600000 +0   2.54500000 +0
  H     1.16900000 +0   0.84100000 +0   5.52300000 +0
  H     1.95100000 +0   2.30000000 +0   5.07900000 +0
  H    -0.21700000 +0  -0.44900000 +0   3.97000000 +0
  H    -0.97200000 +0  -0.74600000 +0   1.61800000 +0
  H    -2.50200000 +0   0.59600000 +0   0.04500000 +0
  H    -3.24800000 +0   0.31600000 +0  -2.30200000 +0
  H    -2.63200000 +0   0.42900000 +0  -4.65700000 +0
  H    -1.04100000 +0   0.59600000 +0  -5.17800000 +0
  H     0.85000000 +0   0.72400000 +0  -3.63500000 +0
  H     1.57300000 +0   1.04200000 +0  -1.28800000 +0
  H     1.03000000 +0   2.91900000 +0   0.47800000 +0
  H     1.77300000 +0   3.26500000 +0   2.82800000 +0
 
