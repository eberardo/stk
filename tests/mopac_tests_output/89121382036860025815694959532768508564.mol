
     RDKit          3D

  0  0  0  0  0  0  0  0  0  0999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 26 27 0 0 0
M  V30 BEGIN ATOM
M  V30 1 N 1.3620 1.5220 4.7960 0
M  V30 2 C 0.8860 1.3970 3.5500 0
M  V30 3 C 0.0620 0.2720 3.1960 0
M  V30 4 C -0.3600 0.1190 1.8970 0
M  V30 5 C 0.0040 1.0550 0.9010 0
M  V30 6 C -0.4220 0.8550 -0.4760 0
M  V30 7 C -1.7730 0.6290 -0.7710 0
M  V30 8 C -2.1930 0.4720 -2.0800 0
M  V30 9 C -1.2510 0.5170 -3.1280 0
M  V30 10 N -1.6550 0.2820 -4.4380 0
M  V30 11 C 0.1150 0.7120 -2.8330 0
M  V30 12 C 0.5150 0.8880 -1.5190 0
M  V30 13 C 0.7770 2.1840 1.2510 0
M  V30 14 C 1.2000 2.3760 2.5450 0
M  V30 15 H 1.1690 0.8410 5.5230 0
M  V30 16 H 1.9510 2.3000 5.0790 0
M  V30 17 H -0.2170 -0.4490 3.9700 0
M  V30 18 H -0.9720 -0.7460 1.6180 0
M  V30 19 H -2.5020 0.5960 0.0450 0
M  V30 20 H -3.2480 0.3160 -2.3020 0
M  V30 21 H -2.6320 0.4290 -4.6570 0
M  V30 22 H -1.0410 0.5960 -5.1780 0
M  V30 23 H 0.8500 0.7240 -3.6350 0
M  V30 24 H 1.5730 1.0420 -1.2880 0
M  V30 25 H 1.0300 2.9190 0.4780 0
M  V30 26 H 1.7730 3.2650 2.8280 0
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 0 1 2 1
M  V30 1 1 3 2
M  V30 2 1 4 3
M  V30 3 1 5 4
M  V30 4 1 6 5
M  V30 5 1 7 6
M  V30 6 1 8 7
M  V30 7 1 9 8
M  V30 8 1 10 9
M  V30 9 1 11 9
M  V30 10 1 12 11
M  V30 11 1 12 6
M  V30 12 1 13 5
M  V30 13 1 14 13
M  V30 14 1 14 2
M  V30 15 1 15 1
M  V30 16 1 16 1
M  V30 17 1 17 3
M  V30 18 1 18 4
M  V30 19 1 19 7
M  V30 20 1 20 8
M  V30 21 1 21 10
M  V30 22 1 22 10
M  V30 23 1 23 11
M  V30 24 1 24 12
M  V30 25 1 25 13
M  V30 26 1 26 14
M  V30 END BOND
M  V30 END CTAB
M  END

$$$$
