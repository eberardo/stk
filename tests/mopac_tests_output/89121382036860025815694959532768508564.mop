PM7 NOOPT  EPS=80.1  CHARGE=1 
89121382036860025815694959532768508564

N   1.362   +1  1.522   +1  4.796   +1 
C   0.886   +1  1.397   +1  3.55   +1 
C   0.062   +1  0.272   +1  3.196   +1 
C   -0.36   +1  0.119   +1  1.897   +1 
C   0.004   +1  1.055   +1  0.901   +1 
C   -0.422   +1  0.855   +1  -0.476   +1 
C   -1.773   +1  0.629   +1  -0.771   +1 
C   -2.193   +1  0.472   +1  -2.08   +1 
C   -1.251   +1  0.517   +1  -3.128   +1 
N   -1.655   +1  0.282   +1  -4.438   +1 
C   0.115   +1  0.712   +1  -2.833   +1 
C   0.515   +1  0.888   +1  -1.519   +1 
C   0.777   +1  2.184   +1  1.251   +1 
C   1.2   +1  2.376   +1  2.545   +1 
H   1.169   +1  0.841   +1  5.523   +1 
H   1.951   +1  2.3   +1  5.079   +1 
H   -0.217   +1  -0.449   +1  3.97   +1 
H   -0.972   +1  -0.746   +1  1.618   +1 
H   -2.502   +1  0.596   +1  0.045   +1 
H   -3.248   +1  0.316   +1  -2.302   +1 
H   -2.632   +1  0.429   +1  -4.657   +1 
H   -1.041   +1  0.596   +1  -5.178   +1 
H   0.85   +1  0.724   +1  -3.635   +1 
H   1.573   +1  1.042   +1  -1.288   +1 
H   1.03   +1  2.919   +1  0.478   +1 
H   1.773   +1  3.265   +1  2.828   +1 
