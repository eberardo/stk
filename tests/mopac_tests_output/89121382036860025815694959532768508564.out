 *******************************************************************************
 ** Site#: 23169        For non-commercial use only     Version 16.175M 64BITS**
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 16.175M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left:  45**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Wed May 10 12:35:59 2017  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  NOOPT      - DO NOT OPTIMIZE ANY COORDINATES
 *
 *
 *
 *                 CHARGE ON SYSTEM =    +1
 *
 *
 *
 *  EPS= 80.10 - USE ANDREAS KLAMT'S COSMO IMPLICIT SOLVATION MODEL
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM7 NOOPT  EPS=80.1  CHARGE=1
 89121382036860025815694959532768508564

   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          1.36200000      1.52200000      4.79600000   
     2       C          0.88600000      1.39700000      3.55000000   
     3       C          0.06200000      0.27200000      3.19600000   
     4       C         -0.36000000      0.11900000      1.89700000   
     5       C          0.00400000      1.05500000      0.90100000   
     6       C         -0.42200000      0.85500000     -0.47600000   
     7       C         -1.77300000      0.62900000     -0.77100000   
     8       C         -2.19300000      0.47200000     -2.08000000   
     9       C         -1.25100000      0.51700000     -3.12800000   
    10       N         -1.65500000      0.28200000     -4.43800000   
    11       C          0.11500000      0.71200000     -2.83300000   
    12       C          0.51500000      0.88800000     -1.51900000   
    13       C          0.77700000      2.18400000      1.25100000   
    14       C          1.20000000      2.37600000      2.54500000   
    15       H          1.16900000      0.84100000      5.52300000   
    16       H          1.95100000      2.30000000      5.07900000   
    17       H         -0.21700000     -0.44900000      3.97000000   
    18       H         -0.97200000     -0.74600000      1.61800000   
    19       H         -2.50200000      0.59600000      0.04500000   
    20       H         -3.24800000      0.31600000     -2.30200000   
    21       H         -2.63200000      0.42900000     -4.65700000   
    22       H         -1.04100000      0.59600000     -5.17800000   
    23       H          0.85000000      0.72400000     -3.63500000   
    24       H          1.57300000      1.04200000     -1.28800000   
    25       H          1.03000000      2.91900000      0.47800000   
    26       H          1.77300000      3.26500000      2.82800000   
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         N        1.3620    1.5220    4.7960
     2         C        0.8860    1.3970    3.5500
     3         C        0.0620    0.2720    3.1960
     4         C       -0.3600    0.1190    1.8970
     5         C        0.0040    1.0550    0.9010
     6         C       -0.4220    0.8550   -0.4760
     7         C       -1.7730    0.6290   -0.7710
     8         C       -2.1930    0.4720   -2.0800
     9         C       -1.2510    0.5170   -3.1280
    10         N       -1.6550    0.2820   -4.4380
    11         C        0.1150    0.7120   -2.8330
    12         C        0.5150    0.8880   -1.5190
    13         C        0.7770    2.1840    1.2510
    14         C        1.2000    2.3760    2.5450
    15         H        1.1690    0.8410    5.5230
    16         H        1.9510    2.3000    5.0790
    17         H       -0.2170   -0.4490    3.9700
    18         H       -0.9720   -0.7460    1.6180
    19         H       -2.5020    0.5960    0.0450
    20         H       -3.2480    0.3160   -2.3020
    21         H       -2.6320    0.4290   -4.6570
    22         H       -1.0410    0.5960   -5.1780
    23         H        0.8500    0.7240   -3.6350
    24         H        1.5730    1.0420   -1.2880
    25         H        1.0300    2.9190    0.4780
    26         H        1.7730    3.2650    2.8280

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C12 H12 N2  =    26 atoms



      MOLECULAR POINT GROUP   :   C2  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =   35
                           NO. OF BETA  ELECTRONS =   34

 -------------------------------------------------------------------------------
PM7 NOOPT  EPS=80.1  CHARGE=1
 89121382036860025815694959532768508564



     NO PARAMETERS MARKED FOR OPTIMIZATION, SO 1SCF WAS USED  
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 16.175M)
                                                       Wed May 10 12:35:59 2017
                                                       No. of days remaining =  45




          FINAL HEAT OF FORMATION =        156.30008 KCAL/MOL =     653.95955 KJ/MOL
          VAN DER WAALS AREA      =        225.62 SQUARE ANGSTROMS


          TOTAL ENERGY            =      -2003.35696 EV
          ELECTRONIC ENERGY       =      -2922.56256 EV  POINT GROUP:     C2  
          CORE-CORE REPULSION     =        919.20560 EV
          DIELECTRIC ENERGY       =         -2.69325 EV
          COSMO AREA              =        225.62 SQUARE ANGSTROMS
          COSMO VOLUME            =        232.49 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =          8.726065 EV
          ALPHA SOMO LUMO (EV)    =         -8.726 -0.971
          BETA  SOMO LUMO (EV)    =         -8.919 -2.595
          NO. OF ALPHA ELECTRONS  =         35
          NO. OF BETA  ELECTRONS  =         34
          MOLECULAR WEIGHT        =        184.2402

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    16    H    21    10.92219
            H    17    H    23     4.77163
            H    26    H    18     2.20942


          SCF CALCULATIONS        =          2
 
          WALL-CLOCK TIME         =          0.039 SECONDS
          COMPUTATION TIME        =          0.160 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          1.36200000      1.52200000      4.79600000   
     2       C          0.88600000      1.39700000      3.55000000   
     3       C          0.06200000      0.27200000      3.19600000   
     4       C         -0.36000000      0.11900000      1.89700000   
     5       C          0.00400000      1.05500000      0.90100000   
     6       C         -0.42200000      0.85500000     -0.47600000   
     7       C         -1.77300000      0.62900000     -0.77100000   
     8       C         -2.19300000      0.47200000     -2.08000000   
     9       C         -1.25100000      0.51700000     -3.12800000   
    10       N         -1.65500000      0.28200000     -4.43800000   
    11       C          0.11500000      0.71200000     -2.83300000   
    12       C          0.51500000      0.88800000     -1.51900000   
    13       C          0.77700000      2.18400000      1.25100000   
    14       C          1.20000000      2.37600000      2.54500000   
    15       H          1.16900000      0.84100000      5.52300000   
    16       H          1.95100000      2.30000000      5.07900000   
    17       H         -0.21700000     -0.44900000      3.97000000   
    18       H         -0.97200000     -0.74600000      1.61800000   
    19       H         -2.50200000      0.59600000      0.04500000   
    20       H         -3.24800000      0.31600000     -2.30200000   
    21       H         -2.63200000      0.42900000     -4.65700000   
    22       H         -1.04100000      0.59600000     -5.17800000   
    23       H          0.85000000      0.72400000     -3.63500000   
    24       H          1.57300000      1.04200000     -1.28800000   
    25       H          1.03000000      2.91900000      0.47800000   
    26       H          1.77300000      3.26500000      2.82800000   

                             CARTESIAN COORDINATES

   1    N        1.362000000     1.522000000     4.796000000
   2    C        0.886000000     1.397000000     3.550000000
   3    C        0.062000000     0.272000000     3.196000000
   4    C       -0.360000000     0.119000000     1.897000000
   5    C        0.004000000     1.055000000     0.901000000
   6    C       -0.422000000     0.855000000    -0.476000000
   7    C       -1.773000000     0.629000000    -0.771000000
   8    C       -2.193000000     0.472000000    -2.080000000
   9    C       -1.251000000     0.517000000    -3.128000000
  10    N       -1.655000000     0.282000000    -4.438000000
  11    C        0.115000000     0.712000000    -2.833000000
  12    C        0.515000000     0.888000000    -1.519000000
  13    C        0.777000000     2.184000000     1.251000000
  14    C        1.200000000     2.376000000     2.545000000
  15    H        1.169000000     0.841000000     5.523000000
  16    H        1.951000000     2.300000000     5.079000000
  17    H       -0.217000000    -0.449000000     3.970000000
  18    H       -0.972000000    -0.746000000     1.618000000
  19    H       -2.502000000     0.596000000     0.045000000
  20    H       -3.248000000     0.316000000    -2.302000000
  21    H       -2.632000000     0.429000000    -4.657000000
  22    H       -1.041000000     0.596000000    -5.178000000
  23    H        0.850000000     0.724000000    -3.635000000
  24    H        1.573000000     1.042000000    -1.288000000
  25    H        1.030000000     2.919000000     0.478000000
  26    H        1.773000000     3.265000000     2.828000000


           Empirical Formula: C12 H12 N2  =    26 atoms



      MOLECULAR POINT GROUP   :   C2  


          ALPHA EIGENVALUES  
 -38.38744 -36.71742 -31.39589 -30.49156 -27.25983 -27.00763 -26.24187 -24.58569
 -22.59904 -22.31562 -21.51103 -20.58001 -19.07524 -18.56267 -18.18096 -17.37238
 -17.00999 -15.64543 -15.54477 -15.40877 -15.20068 -15.08915 -14.86574 -14.61403
 -14.38782 -14.06481 -13.14220 -12.76790 -12.66988 -12.10854 -11.80412 -11.07719
 -10.37920 -10.01214  -8.72607  -0.97089  -0.52351  -0.17648  -0.08583   1.44211
   1.64344   2.10452   2.82165   2.88530   3.06170   3.44095   3.77251   3.98046
   4.00790   4.30386   4.37821   4.56930   4.68496   4.90619   4.97988   5.06397
   5.28113   5.36948   5.43165   5.61794   5.63162   5.68246   5.76723   5.83224
   5.90782   6.04384   6.10969   6.77358


           BETA EIGENVALUES 
 -37.31090 -36.66643 -31.32317 -30.44770 -27.13037 -26.97892 -26.22803 -24.48027
 -22.53709 -22.22626 -21.47351 -20.53777 -18.99005 -18.35025 -18.12530 -17.32318
 -16.94317 -15.51832 -15.38414 -15.18659 -15.09134 -14.97591 -14.65928 -14.43760
 -14.34094 -14.01068 -12.74385 -12.64400 -12.31681 -11.93227 -11.36608 -10.94614
 -10.40605  -8.91924  -2.59523  -0.39406  -0.15877  -0.08168   0.30919   1.69905
   2.05573   2.13638   2.86972   2.90393   3.07894   3.51414   3.78346   4.01541
   4.02733   4.33801   4.42075   4.57840   4.70347   4.99181   5.00752   5.15145
   5.38192   5.43456   5.45335   5.59137   5.62375   5.68340   5.84441   5.92437
   5.95968   6.03080   6.11825   6.79054


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N          -0.203769        5.2038     1.50381     3.69996
    2          C           0.150024        3.8500     1.06896     2.78101
    3          C          -0.137861        4.1379     1.09832     3.03954
    4          C          -0.146759        4.1468     1.09190     3.05486
    5          C           0.073352        3.9266     1.08069     2.84596
    6          C          -0.177037        4.1770     1.05077     3.12627
    7          C          -0.053238        4.0532     1.09033     2.96291
    8          C          -0.309632        4.3096     1.07531     3.23432
    9          C           0.261738        3.7383     1.06360     2.67466
   10          N          -0.619834        5.6198     1.57058     4.04926
   11          C          -0.308711        4.3087     1.07569     3.23302
   12          C          -0.053483        4.0535     1.09089     2.96259
   13          C          -0.147131        4.1471     1.09153     3.05560
   14          C          -0.135736        4.1357     1.09843     3.03730
   15          H           0.346490        0.6535     0.65351
   16          H           0.348934        0.6511     0.65107
   17          H           0.197813        0.8022     0.80219
   18          H           0.193500        0.8065     0.80650
   19          H           0.167175        0.8328     0.83282
   20          H           0.182052        0.8179     0.81795
   21          H           0.316121        0.6839     0.68388
   22          H           0.314868        0.6851     0.68513
   23          H           0.181649        0.8184     0.81835
   24          H           0.167469        0.8325     0.83253
   25          H           0.194111        0.8059     0.80589
   26          H           0.197894        0.8021     0.80211
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     4.582     2.643    13.789    14.769
 HYBRID         0.113     0.610     0.438     0.760
 SUM            4.696     3.253    14.228    15.332


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   1.50381   1.27665   1.27555   1.14776
    2  C   1.06896   0.94054   0.98810   0.85237
    3  C   1.09832   0.97334   1.00901   1.05719
    4  C   1.09190   1.01976   1.05766   0.97744
    5  C   1.08069   0.92000   0.94726   0.97870
    6  C   1.05077   1.00685   1.14919   0.97024
    7  C   1.09033   1.01165   0.91168   1.03958
    8  C   1.07531   1.09366   1.15141   0.98924
    9  C   1.06360   0.96267   0.86124   0.85075
   10  N   1.57058   1.23383   1.68668   1.12875
   11  C   1.07569   1.03017   1.14792   1.05493
   12  C   1.09089   1.07381   0.91598   0.97279
   13  C   1.09153   0.98309   1.03538   1.03713
   14  C   1.09843   1.01319   1.04177   0.98234
   15  H   0.65351
   16  H   0.65107
   17  H   0.80219
   18  H   0.80650
   19  H   0.83282
   20  H   0.81795
   21  H   0.68388
   22  H   0.68513
   23  H   0.81835
   24  H   0.83253
   25  H   0.80589
   26  H   0.80211


                    (SZ)    =    0.500000
                    (S**2)  =    0.963000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  N  0.5944  0.026  0.349  0.157  0.063
    2  C -0.1974 -0.013 -0.101 -0.050 -0.034
    3  C  0.3638  0.011  0.223  0.103  0.028
    4  C -0.2742 -0.010 -0.163 -0.074 -0.027
    5  C  0.4441  0.014  0.278  0.123  0.030
    6  C -0.0996 -0.006 -0.009 -0.071 -0.014
    7  C  0.0882  0.004  0.007  0.073  0.004
    8  C -0.0690 -0.002 -0.004 -0.058 -0.004
    9  C  0.0662  0.002  0.004  0.059  0.001
   10  N  0.0290  0.002  0.001  0.025  0.000
   11  C -0.0688 -0.002 -0.004 -0.059 -0.004
   12  C  0.0880  0.004  0.005  0.070  0.009
   13  C -0.2752 -0.010 -0.170 -0.071 -0.023
   14  C  0.3670  0.011  0.241  0.087  0.029
   15  H -0.0249 -0.025
   16  H -0.0249 -0.025
   17  H -0.0109 -0.011
   18  H  0.0081  0.008
   19  H -0.0027 -0.003
   20  H  0.0022  0.002
   21  H -0.0001 -0.000
   22  H -0.0001 -0.000
   23  H  0.0023  0.002
   24  H -0.0027 -0.003
   25  H  0.0082  0.008
   26  H -0.0110 -0.011

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.05 SECONDS

 == MOPAC DONE ==
